Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -55,7 +55,10 @@ def audit_desc(type, name, desc_call)
         end
 
         # Check if the desc starts with the formula's or cask's name.
-        problem "Description shouldn't start with the #{type} name." if regex_match_group(desc, /^#{name} /i)
+        name_regex = name.delete("-").split("").join('[\s\-]?')
+        if regex_match_group(desc, /^#{name_regex}(?=\s)/i)
```

Comment:
How about checking for a word boundary instead?

```suggestion
        if regex_match_group(desc, /^#{name_regex}\b/i)
```